docs/src/components/ColorSwatch/ColorSwatch.component.tsx

```diff
@@ -0,0 +1,104 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import styled, { css } from 'styled-components';
+import classNames from 'classnames';
+import { Pill } from '../Pill'
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+// THEME
+import { colors, sizes } from '../../../../src/theme';
+
+
+// INTERFACES
+interface ColorSwatchProps {
+  className?: string;
+  backgroundColor: string;
+  contrastOnBlack: number;
+  contrastOnF1F1F1: string;
+  contrastOnFAFAFA: string;
+  contrastOnWhite: number;
+  copyChoice: string;
+  name: string;
+  size: 'large' | 'small';
+  textColor: string;
+}
+
+interface ColorSwatchStylesGroup {
+  [key: string]: FlattenSimpleInterpolation;
+}
+
+
+// STYLES
+const ColorSwatchSizeStyles: ColorSwatchStylesGroup = {
+  large: css`
+    .ColorSwatchPillContainer {
+      padding-top: 200px
+    }
+  `,
+  small: css`
+    .ColorSwatchPillContainer {
+      padding-top: 50px
+    }
+  `,
+}
+
+const StyledColorSwatch = styled.div`
+  background: ${props => props.backgroundColor };
+  color: ${props => props.textColor };
+  padding: 24px 16px 16px 16px;
```

Comment:
We can use `REM`s here (1rem = 16px) and we can abbreviate this CSS: `padding 1.5rem 1rem 1rem;`